understanding/documenting-accessibility-support.html

```diff
@@ -5,7 +5,7 @@
 	</head>
 	<body>
 		<h1>Documenting Accessibility Support for Uses of a Web Technology</h1>
-		<p>The documentation of accessibility support for uses of a Web technology provides the information needed to determine whether it is possible to satisfy the WCAG 2.1 Success Criteria for a particular environment. </p>
+		<p>The documentation of accessibility support for uses of a Web technology provides the information needed to determine whether it is possible to satisfy the WCAG Success Criteria for a particular environment. </p>
```

Comment:
Since we're working on WCAG 3, I think it would be better to keep a number in.

```suggestion
		<p>The documentation of accessibility support for uses of a Web technology provides the information needed to determine whether it is possible to satisfy the WCAG 2.2 Success Criteria for a particular environment. </p>
```

Reply:
Hi @WilcoFiers I think this PR now successfully switches any generic references to be `WCAG 2` and a few very specific WCAG 2.2 references to be `WCAG 2.2`. Would you mind reviewing one final time to unblock the PR?

Reply:
I believe what he has flagged has been resolved. @WilcoFiers I'll be marking this review as resolved and merging this today, unless you respond.
If you believe something needs to be resolved after the merge, we will prioritize review of it.